proto/greenfield/payment/params.proto

```diff
@@ -26,3 +25,9 @@ message Params {
     (gogoproto.nullable) = false
   ];
 }
+
+// VersionedParams defines the parameters with multiple versions, each version is stored with different timestamp.
+message VersionedParams {
+  // Time duration which the buffer balance need to be reserved for NetOutFlow e.g. 6 month
+  uint64 reserve_time = 1 [(gogoproto.moretags) = "yaml:\"reserve_time\""];
```

Comment:
`validator_tax_rate` should also be put into versioned params

Reply:
added it